packages/generator/templates/operations.ts.hbs

```diff
@@ -34,9 +34,9 @@
         },
         retrySettings?: OperationOptions,
         headers?: { [key: string]: string }{{#or (is method "patch") (is method "post") (is method "put")}},
-        body: {{{getRequestPayloadType request}}}{{/or}}
+        body: {{{getRequestPayloadTypeWithNamespace request}}}{{/or}}
```

Comment:
Another option is to have this be 2 helpers.  One that calls another within this template.  Something like `{{{addNamespace (getRequestPayloadType request)}}}` This would also help to ensure DRYer code for this process.